src/CMakeLists.txt

```diff
@@ -209,15 +211,15 @@ target_link_libraries(example libsoplex)
 set_target_properties(soplex PROPERTIES INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
```

Comment:
should `lib` also be replaced here?

Reply:
Yes, I think so. I will push a follow-up commit.

Reply:
This didn’t cause a problem in my testing for a potential Fedora Linux package because I was already [patching out rpaths](https://docs.fedoraproject.org/en-US/packaging-guidelines/#_beware_of_rpath).